.travis.yml

```diff
@@ -35,8 +35,13 @@ before_install:
   - uname -r
 
 install:
-  - wget https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip -O /tmp/protoc-3.3.0-linux-x86_64.zip
-  - sudo unzip -o -d /usr/local /tmp/protoc-3.3.0-linux-x86_64.zip
+  - wget https://github.com/google/protobuf/releases/download/v3.5.0/protoc-3.5.0-linux-x86_64.zip -O /tmp/protoc-3.5.0-linux-x86_64.zip
+  - sudo unzip -o -d /usr/local /tmp/protoc-3.5.0-linux-x86_64.zip
+  - sudo chmod +x /usr/local/bin/protoc
+  - sudo chmod og+rx /usr/local/include/google/protobuf
```

Comment:
just question: why this became needed now?

Reply:
They changed the default perms in the zip file. I am still trying to figure out why these aren't readable.